spec/lib/vcr/errors_spec.rb

```diff
@@ -56,7 +56,7 @@ def request_with(values)
         end
       end
 
-      context 'when there is a current cassette' do
+      context 'when there are cassettes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.